src/com/sap/piper/WhitesourceConfigurationHelper.groovy

```diff
@@ -86,7 +86,7 @@ class WhitesourceConfigurationHelper implements Serializable {
                 mapping += [
                     [name: 'docker.scanImages', value: true, force: true],
                     [name: 'docker.scanTarFiles', value: true, force: true],
-                    [name: 'docker.includes', value: '*.tar', force: true],
+                    [name: 'docker.includes', value: '.*\.tar', force: true],
```

Comment:
May be `*` has to be escaped like this `\\*`?  I am also getting the same error.
Ref [here](https://stackoverflow.com/questions/40246231/java-util-regex-patternsyntaxexception-dangling-meta-character-near-index-0).

Reply:
I missed to escape the `\` in the first commit I pushed. But now `.*\\.tar` seems to work.

Reply:
Why do we need first occurrence of `.`? Couldn't  we have simply `*\\.tar`?

Reply:
I think it has to be a regular expression.